packages/http-client-java/generator/http-client-generator-test/src/main/java/com/cadl/multipart/models/InheritFileData.java

```diff
@@ -9,9 +9,7 @@
 import com.azure.core.util.BinaryData;
 
 /**
- * Image file of content-type "image/jpeg"
- * 
- * The file details for the "fileData1" field.
```

Comment:
I thought this line "The file details for the "fileData1" field." is added in emitter (not by TCGC).

Could you check if the logic is `add the "The file details ..." description when there was no description`? If this is the logic, we should be good.

Reply:
I looked more into the logics we have. 
We will add the `"The file details ..."` in [`createFileDetailsSchema()`](https://github.com/microsoft/typespec/blob/main/packages/http-client-java/emitter/src/external-schemas.ts#L168), and we overwrite schema's `summary` and `description` [here](https://github.com/microsoft/typespec/blob/main/packages/http-client-java/emitter/src/external-schemas.ts#L276-L290)
Previously  `description` is  'Image file of content-type `image/jpeg"` and we put `description` to schema's `summary`, so the schema's `description` is not overwriten. But now we put `doc` to schema's `description`, so the schema's  `description` is overwriten
![image](https://github.com/user-attachments/assets/fa70a840-842d-4d21-84ae-92d9f959c7f4)

In summary, the logic is basically use TCGC's `doc` to replace "The file details ..." if TCGC has `doc`.
If we want to keep both, we can append TCGC's `doc` to schema's description.

Reply:
Got it.

Yes, behavior in your PR is what we expected.

`createFileDetailsSchema` only fill the default doc (and apply in case of e.g. `bytes` that does not have a model at all). And when we had a File schema, we'd use its doc/summary.